tests/unit/spec/base/mozilla-monitor-button.js

```diff
@@ -40,14 +40,14 @@ describe('mozilla-monitor-button.js', function() {
     it('should make a single metrics flow request', function() {
         spyOn(window, 'fetch').and.callThrough();
 
-        return Mozilla.MonitorButton.init().then(function() {
+        return Mozilla.FxaProductButton.init().then(function() {
             expect(window.fetch).toHaveBeenCalledTimes(1);
             expect(window.fetch).toHaveBeenCalledWith('https://accounts.firefox.com/metrics-flow?entrypoint=mozilla.org-firefox-accounts&form_type=button&utm_source=mozilla.org-firefox-accounts&utm_campaign=trailhead');
         });
     });
 
     it('should attach flow parameters to button hrefs in the metrics response', function() {
-        return Mozilla.MonitorButton.init().then(function() {
+        return Mozilla.FxaProductButton.init().then(function() {
             var buttons = document.querySelectorAll('.js-monitor-button');
             expect(buttons[0].href).toEqual('https://monitor.firefox.com/oauth/init?form_type=button&entrypoint=mozilla.org-firefox-accounts&utm_source=mozilla.org-firefox-accounts&utm_campaign=trailhead&utm_medium=referral&deviceId=848377ff6e3e4fc982307a316f4ca3d6&flowBeginTime=1573052386673&flowId=75f9a48a0f66c2f5919a0989605d5fa5dd04625ea5a2ee59b2d5d54637c566d1');
```

Comment:
Since we are moving to the more generic `FxaProductButton`, should these tests be flexible enough to cover other product hrefs such as `https://getpocket.com/...`?